src/modules/jobs.rs

```diff
@@ -15,12 +15,24 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         .trim()
         .parse::<i64>()
         .ok()?;
-    if num_of_jobs == 0 {
+
+    if config.threshold < 0 {
+        log::warn!(
+            "threshold in [jobs] ({}) was less than zero",
+            config.threshold
+        );
+        return None;
+    }
+
+    if num_of_jobs == 0 && config.threshold > 0 {
         return None;
     }
 
     let module_number = if num_of_jobs > config.threshold {
         num_of_jobs.to_string()
+    } else if num_of_jobs == 0 && config.threshold == 0 {
+        // Explicitly show when there are 0 background jobs
+        "0".to_string()
```

Comment:
You don't need separate branch. If threshold is zero number will always get printed.
```suggestion
    let module_number = if num_of_jobs > config.threshold || config.threshold == 0 {
        num_of_jobs.to_string()
    } else {
        "".to_string()
```